rainforest/runner.go

```diff
@@ -39,6 +39,8 @@ type RunStatus struct {
 		Total    int `json:"total"`
 		Complete int `json:"complete"`
 		NoResult int `json:"no_result"`
+		Passed   int `json:"passed"`
+		Failed   int `json:"failed"`
```

Comment:
These two are passed by default, even if there are no tests with that state - https://github.com/rainforestapp/Rainforest/blob/85da6c4424376b7a85d72106c3d98d386a3707c3/app/models/run.rb#L608